src/Test/Wordpress/WordPressAPITest.php

```diff
@@ -1,34 +0,0 @@
-<?php
```

Comment:
Initially, I removed it and deleted the test cause we didn't need it. But I reverted the commit. I'll add the tests back again.